public/app/core/components/AppChrome/MegaMenu/MegaMenu.tsx

```diff
@@ -129,7 +129,7 @@ export const MegaMenu = memo(
             </ul>
           </ScrollContainer>
           {shouldRenderInviteUserButton && (
-            <div className={styles.inviteNewMemberButton}>
+            <div className={styles.inviteNewUserButton}>
```

Comment:
should we keep the megamenu option in addition to the top nav button? they dont conflict IMO and doesn't hurt but interested in team's thoughts